app/javascript/components/Results/Results.tsx

```diff
@@ -14,29 +16,37 @@ interface ResultsProps {
 
 export const Results = (props: ResultsProps) => {
   return (
-    <div id="serp-results-wrapper">
-      <div id="results">
-        {props.results.map((result, index) => {
-          return (
-            <div className='result' key={index}>
-              <GridContainer>
-                <Grid row>
-                  <Grid tablet={{ col: true }}><a href= "#" ><h4>{result.title}</h4></a></Grid>
-                </Grid>
-                <Grid row>
-                  <Grid tablet={{ col: true }}><a href= "#">{result.url}</a></Grid>
-                </Grid>
-                {props.vertical === 'image' && <Grid row>
-                  <Grid tablet={{ col: true }}><img src={result.thumbnail.url} className="result-image"/></Grid>
-                </Grid>}
-                <Grid row>
-                  <Grid tablet={{ col: true }}><p>{result.description}</p></Grid>
-                </Grid>
-              </GridContainer>
-            </div>
-          );
-        })}
+    <>
+      <div id="serp-results-wrapper">
+        <div id="results">
+          {props.results.map((result, index) => {
+            return (
+              <div className='result' key={index}>
+                <GridContainer>
+                  <Grid row>
+                    <Grid tablet={{ col: true }}><a href= "#" ><h4>{result.title}</h4></a></Grid>
+                  </Grid>
+                  <Grid row>
+                    <Grid tablet={{ col: true }}><a href= "#">{result.url}</a></Grid>
+                  </Grid>
+                  {props.vertical === 'image' && <Grid row>
+                    <Grid tablet={{ col: true }}><img src={result.thumbnail.url} className="result-image"/></Grid>
+                  </Grid>}
+                  <Grid row>
+                    <Grid tablet={{ col: true }}><p>{result.description}</p></Grid>
+                  </Grid>
+                </GridContainer>
+              </div>
+            );
+          })}
+        </div>
       </div>
-    </div>
+      {props.results.length > 0 && 
+        <Pagination 
+          totalPages={props.results.length}
```

Comment:
I think the `totalPages` value should be different initially, maybe a fixed value until we get pagination data from the backend. What do you think?

Reply:
This sounds good to me. Let me update.

Reply:
@ddzz updated.